language/German/strings.po

```diff
@@ -2152,12 +2196,12 @@ msgstr "INIs"
 #: src/frontend/mame/ui/dirmenu.cpp:54
 msgctxt "path-option"
 msgid "UI Settings"
-msgstr ""
+msgstr "Benutzeroberfläche-Einstellungen"
 
 #: src/frontend/mame/ui/dirmenu.cpp:55
 msgctxt "path-option"
 msgid "Plugin Data"
-msgstr ""
+msgstr "Erweiterungs-Daten"
```

Comment:
"Erweiterungsdateien"

Reply:
No, it's data for the plugins. So it's rather _Daten für Erweiterungen_